packages/web-twig/src/Resources/components/TextFieldBase/README.md

```diff
@@ -39,22 +56,23 @@ Without lexer:
-| `id`              | `string`                                                      | —       | yes      | Input and label identification                             |
-| `isDisabled`      | `bool`                                                        | `false` | no       | If true, input is disabled                                 |
-| `isFluid`         | `bool`                                                        | `false` | no       | If true, the element spans to the full width of its parent |
-| `isLabelHidden`   | `bool`                                                        | `false` | no       | If true, label is hidden                                   |
-| `isMultiline`     | `bool`                                                        | `false` | no       | If true, rendered DOM element is `textarea`                |
-| `isRequired`      | `bool`                                                        | `false` | no       | If true, input is required                                 |
-| `label`           | `string`                                                      | —       | yes      | Label text                                                 |
-| `message`         | `string`                                                      | `null`  | no       | Validation or help message                                 |
-| `name`            | `string`                                                      | `null`  | no       | Input name                                                 |
-| `placeholder`     | `string`                                                      | `null`  | no       | Input placeholder                                          |
-| `type`            | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | `text`  | no       | Input type                                                 |
-| `validationState` | `success`, `warning`, `error`                                 | `null`  | no       | Type of validation state                                   |
-| `value`           | `string`                                                      | `null`  | no       | Input value                                                |
+| Prop name           | Type                                                          | Default | Required | Description                                                             |
+| ------------------- | ------------------------------------------------------------- | ------- | -------- | ----------------------------------------------------------------------- |
+| `class`             | `string`                                                      | `null`  | no       | Custom CSS class                                                        |
+| `hasPasswordToggle` | `bool`                                                        | `false` | no       | If true, the `type` is set to `password` and there is a password toggle |
```

Comment:
My proficiency in English is quite poor, but I feel like the sentences are better "connected" like this: 

```suggestion
| `hasPasswordToggle` | `bool`                                                        | `false` | no       | If true, the `type` is set to `password` and a password toggle is shown |
```